### PR TITLE
finaly

### PR DESCRIPTION
**Pull request by hoanq-2556**
https://edu-redmine.sun-asterisk.vn/issues/37965

### CODE REVIEW

**Comment by hoanq-2556** [comment]
https://user-images.githubusercontent.com/85326558/125556609-4bc7187a-3019-4e5d-ba2a-81b7a6edf503.png
https://user-images.githubusercontent.com/85326558/125556654-23bad301-e53b-43bf-b94b-2276a2b8a3b6.png

**Comment by hoanq-2556** [comment]
image
image

**Review by luongvanlinh — APPROVED** [review]

**Comment by hoanq-2556** [comment]
Ready

File: app/controllers/course_users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class CourseUsersController < ApplicationController
+  before_action :load_course_user, only: [:show]
+
+  def index
+    @course_users = current_user.course_users.page(params[:page]).per(Settings.page.per)
```
Comment (tham-1781) [inline app/controllers/course_users_controller.rb]:
```suggestion
    @course_users = current_user.course_users.includes(:course).page(params[:page]).per(Settings.page.per)
```

File: app/controllers/course_users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    @course_users = @course_users.includes :course
```
Comment (tham-1781) [inline app/controllers/course_users_controller.rb]: xóa line này đi

File: app/controllers/course_users_controller.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+  def show
+    @course = @course_user.course
+  end
+
+  private
+
+  def load_course_user
+    @course_user = current_user.course_users.find_by id: params[:id]
+    return if @course_user.present?
```
Comment (tham-1781) [inline app/controllers/course_users_controller.rb]:
```suggestion
    return if @course_user
```

**Comment by hoanq-2556** [comment]
Ready

**Review by tham-1781 — APPROVED** [review]

**Pull request merged**